.circleci/config.yml

```diff
@@ -152,12 +152,12 @@ commands:
     steps:
       - restore_cache:
           name: Restore sccache cache
-          key: sccache-cache-stable-{{ .Branch }}-{{ .Environment.CIRCLE_TAG }}
+          key: sccache-cache-stable-{{ .Branch }}-{{ .Environment.CIRCLE_JOB }}
```

Comment:
We don't currently use these sccache steps at all (I was tempted to remove them completely). If it was enabled I don't think `CIRCLE_TAG` does much of anything for restoring the most appropriate version of the sccache build artifacts. So I copied `CIRCLE_JOB` from [application-services](https://github.com/mozilla/application-services/blob/f14d3ab/.circleci/config.yml#L169) mostly just to avoid any confusion with `CIRCLE_TAG`.